integration-tests/utils/integration-helpers.js

```diff
@@ -57,6 +57,21 @@ const createEntityInWorkspace = (page, billingProject, workspaceName, testEntity
   }, billingProject, workspaceName, testEntity)
 }
 
+const checkBucketAccess = async (page, billingProject, workspaceName, accessLevel = 'OWNER') => {
+  const details = await page.evaluate((billingProject, workspaceName) => {
+    return window.Ajax().Workspaces.workspace(billingProject, workspaceName).details()
+  }, billingProject, workspaceName)
+  const bucketName = details.workspace.bucketName
+  console.info(`Checking workspace access for ${billingProject}, ${workspaceName}, ${bucketName}.`)
+  // Try polling for workspace bucket access to be available.
+  await page.waitForFunction(async (billingProject, workspaceName, bucketName, accessLevel) => {
+    try {
+      await window.Ajax().Workspaces.workspace(billingProject, workspaceName).checkBucketAccess(billingProject, bucketName, accessLevel)
+      return true
+    } catch (e) { return false }
+  }, { timeout: 60000, polling: 10000 }, billingProject, workspaceName, bucketName, accessLevel)
```

Comment:
```suggestion
  }, { timeout: 60000, polling: 500 }, billingProject, workspaceName, bucketName, accessLevel)
```
Assuming the unit is milliseconds, 10 seconds is a long time IMO. If the bucket becomes available at 3 seconds, we can and should save the users those 7 seconds.

Reply:
It is a 10s polling interval. My rational was that this is something that apparently can take quite a long time to grant (given that the test does several other things before launching the workflow, and yet sometime the access is sometimes not resolved during that time). So I was going for fairly coarse polling intervals and a longer-than-usual timeout to avoid flooding the log with tons of API calls. But sure, we can cut it to 5 seconds.